packages/cf-component-box/src/Box.js

```diff
@@ -0,0 +1,117 @@
+import React, { PropTypes } from 'react';
+import { createComponent } from 'cf-style-container';
+
+const styles = (props) => ({
+  margin: props.margin || props.theme['margin'],
+  marginTop: props.marginTop || props.theme['marginTop'],
+  marginBottom: props.marginBottom || props.theme['marginBottom'],
+  marginLeft: props.marginLeft || props.theme['marginLeft'],
+  marginRight: props.marginRight || props.theme['marginRight'],
+
+  padding: props.padding || props.theme['padding'],
```

Comment:
This will lead to bugs if you define both `padding` and `paddingTop` at the same time. Fela can't enforce the order of properties because of [atomic CSS design](http://fela.js.org/docs/introduction/Drawbacks.html). We can't mix longhand and shorthand properties. It needs to be one of either.